tasks/12-ibm-codeengine-deploy.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: ibm-codeengine-deploy
+spec:
+  params:
+    - name: image-to
+      type: string
+    - name: tools-image
+      default: garagecatalyst/ibmcloud-dev:1.1.3
```

Comment:
We have a newer version of this image hosted in quay.io. Can you update this to `quay.io/ibmgaragecloud/ibmcloud-dev:v2.0.4`?